client/components/user-profile.js

```diff
@@ -0,0 +1,307 @@
+import React from "react";
+import { connect } from "react-redux";
+import { useState, useEffect } from "react";
+import { BlockPicker } from "react-color";
+import {
+  Input,
+  Button,
+  Divider,
+  Icon,
+  Card,
+  Image,
+  List,
+  Form,
+  TextArea,
+} from "semantic-ui-react";
+
+export const UserProfile = () => {
+  const [currentBio, SetCurrentBio] = useState("");
+  const [guiltyPleasure, setGuiltyPleasure] = useState("");
+  const [currentMoodImprovement, setCurrentMoodImprovement] = useState({
```

Comment:
I wanted to create a state, to hold the list items. I wasn't sure if it should be an array or objects. I was going to map them so they render each time one is added.